Unit.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <string>
+class Unit
+{
+private:
+	int hp;
+	int dmg;
+	std::string name;

```

Comment:
const?

Reply:
A health értékét nem gondoljuk, hogy jó ötlet lenne constként beállítani, ugyanis annak értéke a program futása során állandó jelleggel változik.
A damage és name változókat const-ra állítottuk az új verzióban, mivel azok valóban nem változnak a program futása során.

Reply:
Pontosan.